server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -99,21 +100,32 @@ public Content render(
             .getBundle(request)
             .setTitle("CiviForm")
             .addMainContent(
-                renderHeader(tiGroup.getName(), "py-12", "mb-0", "bg-gray-50"),
-                hr().withClasses("mt-6"),
+                h1(tiGroup.getName()).withClasses("py-12", "px-20", "ti-header-band"),
                 div(
-                        renderSubHeader(messages.at(MessageKey.TITLE_ALL_CLIENTS.getKeyName())),
+                        renderSubHeader(messages.at(MessageKey.TITLE_ALL_CLIENTS.getKeyName()))
+                            .withClass("mb-0"),
                         renderAddNewClientButton(messages, tiGroup.id))
-                    .withClasses("flex", "justify-between", "my-4"),
-                h4("Search"),
-                renderSearchForm(request, searchParameters, messages, managedAccounts.size()),
-                renderTIClientsList(
-                    managedAccounts, searchParameters, page, totalPageCount, messages),
-                hr().withClasses("my-6"),
-                renderSubHeader(messages.at(MessageKey.HEADER_ACCT_SETTING.getKeyName())),
-                h3(messages.at(MessageKey.TITLE_ORG_MEMBERS.getKeyName())).withClass("mt-8"),
-                renderTIMembersTable(tiGroup, messages))
-            .addMainStyles("px-20", "max-w-screen-xl");
+                    .withClasses(
+                        "flex",
+                        "justify-between",
+                        "items-center",
+                        "py-4",
+                        "px-20",
+                        "ti-header-band",
+                        "mb-10"),
+                div(
+                        h4("Search"),
```

Comment:
I think there's a localized message for this `Search`, but you can totally do it in a different PR.

Reply:
Could callout.  I'm planning on doing a final check that all text is translated as part of this issue:  #7046